Plugin/Gateway.php

```diff
@@ -57,10 +60,12 @@ public function createPayment($data) {
 
 		try {
 			$response = $this->client->execute();
+			
+			if ($response['return'] === self::CLIENTRESPONSE_APIKEYINVALID) $response['errors'] = 'true';
 
-			return $response['return'];
+			return $response;
 		} catch (\Exception $exception) {
-			return false;
+			return array('errors' => 'true', 'return' => 'Invalid return from the API');
```

Comment:
I would prefer short syntax arrays: `['errors' => 'true', 'return' => 'Invalid return from the API']`